packages/ODEs/README.md

```diff
@@ -0,0 +1,168 @@
+  * shows how Datagrok solves complicated ODEs
+* `Fermentation`
+  * models the kinetics of the biochemical reactions in [fermentation](https://en.wikipedia.org/wiki/Fermentation)
+  * shows the usage of the `runOnOpen` and  `runOnInput` meta-features
+* `PK-PD`
+  * simulates pharmacokinetics (PK), pharmacodynamics (PD), and their [relationship](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7348046)
+  * illustrates the usage of the `loop` feature for dosing specification
+* `Acid production`
+  * models gluconic acid [production](https://oatao.univ-toulouse.fr/9919/1/Elqotbi_9919.pdf) by Aspergillus niger
+  * shows the usage of the `update` feature for multi-stage simulation
+* `Nimotuzumab`
+  * models population pharmacokinetic for [nimotuzumab](https://www.mdpi.com/1999-4923/12/12/1147)
+  * demonstrates the `output` feature
+
+Datagrok's ODEs suite has tools for solving both [stiff](https://en.wikipedia.org/wiki/Stiff_equation) and non-stiff equations. It provides a [numerical solution](https://en.wikipedia.org/wiki/Numerical_methods_for_ordinary_differential_equations).
+
+### Project structure
+
+A project defines [initial value problem](https://en.wikipedia.org/wiki/Initial_value_problem). It contains *name*, *differential equations*, *initial values* and *argument* specifications:
+
+```python
+#name: My ODEs
+#equations:
+  dx/dt = x + y + exp(t)
+  dy/dt = x - y - cos(t)
+#argument: t
+  initial = 0
+  final = 1
+  step = 0.01
+#inits:
+  x = 2
+  y = 5
+```
+
+You can create extensions by defining components in the control blocks.
+
+Find useful the following:
+
+|Control block|Features|
+|-------------|--------|
+|```#name```|Defines a name.|
+|```#equations```|Differential equation specification|
+|```#argument```|Independent variable specification|
+|```#inits```|Initial values specification|
+|```#constants```|Constants specification|
+|```#parameters```|Parameters specification|
+|```#expressions```|Defines auxiliary compuations.|
+|```#output```|Defines output columns and their captions|
+|```#tags```|Specifies tags (`model`, `app`, etc.).|
+|```#description```|Defines description of the model.|
+|```#meta.runOnOpen```|Provides computations immediately upon model launch after the project is exported to JavaScript script.|
+|```#meta.runOnInput```|Updates results immediately upon input changes after the project is exported to JavaScript script.|
+
+### Platform applications
+
+Datagrok provides an export of each solver project to JavaScript script:
+
+* Press **Export** button on the top panel
+* Press **SAVE** button
+
+Find the created script in the platform `Scripts`.
+
+Improve usability. Use `#tags: model` to add your model to `Model Catalog`. Provide a description in the `#description`-line:
+
+```python
+#name: Bioreaction
+#tags: model
+#description: Complex bioreaction simulation
+```
+
+Apply annotations in your project to get better UI. Append them, when defining *parameters*, *initial values* and *argument*.
```

Comment:
You have to link to this article somewhere that describes parameter annotations in details:

https://datagrok.ai/help/datagrok/concepts/functions/func-params-annotation#parameter-types-and-options

Reply:
Done! 
I've added link to the word "annotations", 
and the link "Function annotations" at the end of this document.